### PR TITLE
Added payment related functionalities to Commerce Module #483

### PR DESCRIPTION
**Pull request by srivastava-yash**
Added the three mentioned functionalities in issue #483:

- paymentId
- paymentType
- paymentProvider

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -5,3 +5,6 @@
 	path = externals/fmt
 	url = https://github.com/fmtlib/fmt.git
 
+[submodule "externals/faker-cxx"]
```
Comment (srivastava-yash) [inline .gitmodules:8]: I am not sure of this change, should I remove this?

Comment (cieslarmichal) [reply .gitmodules:8]: Yes, please remove

**Review by cieslarmichal — CHANGES_REQUESTED** [review]
Good job, only to remove git submodule and we are good to merge :)

**Review by cieslarmichal — APPROVED** [review]

**Comment by cieslarmichal** [comment]
You need to remove externals/faker-cxx directory

**Review by cieslarmichal — APPROVED** [review]

**Pull request merged**